pkg/aws/ec2/api/cleanup/eni_cleanup.go

```diff
@@ -149,7 +151,7 @@ func (e *ENICleaner) DeleteLeakedResources() error {
 				_, err := e.EC2Wrapper.DeleteNetworkInterface(&ec2.DeleteNetworkInterfaceInput{
 					NetworkInterfaceId: nwInterface.NetworkInterfaceId,
 				})
-				if err != nil {
+				if err != nil && !strings.Contains(err.Error(), ec2Errors.NotFoundInterfaceID) { // ignore InvalidNetworkInterfaceID.NotFound error
```

Comment:
nit: if we not skip `NotFound`, the following log will be kind of misleading in case the ENI is not found.

Reply:
Actually I think we should skip `NotFound` instead of continue to log them as success. `NotFound` could be caused by various reasons. I don't think we should log `NotFound` as success here.

Reply:
Makes sense, let me update the log.